content/docs/command-reference/repro.md

```diff
@@ -95,6 +91,39 @@ same time (e.g. in separate terminals). After both finish successfully, you can
 then run `dvc repro train`: DVC will know that both branches are already
 up-to-date and only execute the final stage.
 
+### `targets` argument (optional)
```

Comment:
we complicated it way too much, let's simplify instead and make it part of the options sections like we agreed please

Reply:
As you prefer. I put it in Options now but I don't think it makes sense.

Reply:
it doesn't make sense if we make it very long, putting it into the arguments/options force you to keep it short more or less and this is good to my mind. It's better to have a few simple examples and a few words about them then asking user to read a long spec (that might exist, but should be complimentary)

Reply:
also, I doubt that user will care much about it's specific location (like "it looks bad because it's technically not an option " - I doubt that someone will think like this). 

I agree, though that it's better to revisit it, that's why we created a ticket - to support them properly

Reply:
OK, I simplified `targets` and `--glob` (which I put right after targets in the Options list).